src/components/TodoCount.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+const TodoCount = props => {
```

Comment:
```suggestion
const TodoCount = ({count, onChangeViewMode}) => {
```
하위 컴포넌트에 파라미터를 넘길 때 `props`를 사용하셨는데요, 저는 [구조분해할당](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)을 사용하는 것을 선호합니다. 이유는 
1. Intelij의 지원을 받을 수 있습니다. 위와 같이 코드를 수정할 경우 Intelij에서 자동완성으로 파라미터를 잡아줍니다.
1. 해당 컴포넌트를 다른 곳에서 사용할 때, 어떤 파라미터를 넘겨야하는지 조금 더 분명하게 알 수 있습니다.
1. 모든 파라미터 앞에 불필요한 `props.~`가 들어가지 않아도 돼서 코드가 조금 더 간결해집니다.

하지만 이 부분도 개인의 취향이 들어가는 부분인 것 같네요. 이렇게도 작성할 수 있다고 알아두시면 될 것 같습니다!

Reply:
구조분해할당을 사용하니 코드가 간결해졌네요 👏 코드를 작성할 때 활용했으면 아주 유용했을 것 같아요! 다음 미션에는 처음부터 적용해보도록 할게요~ 😄